src/connectors/hits-per-page/connectHitsPerPage.ts

```diff
@@ -63,25 +41,23 @@ export type HitsPerPageConnectorParams = {
   /**
    * Function to transform the items passed to the templates.
    */
-  transformItems?: (
-    objects: HitsPerPageConnectorParamsItem[]
-  ) => HitsPerPageConnectorParamsItem[];
+  transformItems?: <TItem extends HitsPerPageConnectorParamsItem>(
+    objects: TItem[]
+  ) => TItem[];
 };
 
-export type HitsPerPageRendererOptions<
-  THitsPerPageWidgetParams extends HitsPerPageConnectorParams = HitsPerPageConnectorParams
-> = {
+export type HitsPerPageRendererOptions = {
   /**
    * Array of objects defining the different values and labels.
    */
-  items: HitsPerPageRendererOptionsItem[];
+  items: Array<HitsPerPageConnectorParamsItem & { isRefined: boolean }>;
```

Comment:
The connector and renderer items are not the same. It's a coincidence that they've a similar shape. The ones passed to the renderer don't have the `default` property. Here we should avoid reusing the connector type and create another `HitsPerPageRendererParamsItem` type.